test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1

```diff
@@ -158,7 +158,7 @@ Describe "PowerShellGet - Module tests (Admin)" -tags @('Feature', 'RequireAdmin
         Remove-InstalledModules
     }
 
-    ## Marked as 'Pending' on Linux for now because the test requires root privilege but we cannot do it now in our Travis CI Linux build
+    ## Marked as 'Pending' on Linux for now because the test requires root privilege but we cannot do it now in our Azure DevOps Linux build
     It "Should install a module correctly to the required location with AllUsers scope" -Pending:$IsLinux {
```

Comment:
Plan was to remove the pending here and below? Yes?

Reply:
Yes however it needs root.  Aditya said to add tags but I'm not sure how right here so I just reverted it for now.

Reply:
Fixed!